app/lib/composegenerator/v1/utils/networking.py

```diff
@@ -13,7 +13,7 @@ def getFreePort(networkingFile: str, appId: str):
     # Ports used currently in Citadel
     # TODO: Update this list, currently it's outdated
     usedPorts = [80, 433, 443, 8333, 8332, 28332, 28333, 28334, 10009, 8080, 50001, 9050, 3002, 3000, 3300, 3001, 3004, 25441,
-                 3003, 3007, 3006, 3009, 3005, 8898, 3008, 8081, 8082, 8083, 8085, 2222, 8086, 8087, 8008, 8088, 8089, 8091]
+                 3003, 3007, 3006, 3009, 3010, 3005, 8898, 3008, 8081, 8082, 8083, 8085, 2222, 8086, 8087, 8008, 8088, 8089, 8091]
```

Comment:
@pwltr  Just FYI: I'll remove port 3010 soon, this list should only contain public ports. Port 3010 should not be exposed to the outside, only port 8000 (for nginx), which I'll add soon